packages/workbox-background-sync/test/browser/background-sync-queue-plugin.js

```diff
@@ -12,19 +12,19 @@
  */
 
 /* eslint-env mocha, browser */
-/* global chai, workbox */
+/* global expect, workbox */
```

Comment:
`expect` is in the project [ESLint config](https://github.com/GoogleChrome/workbox/blob/master/.eslintrc.js#L27), so it doesn't need to go here (or in any of the other test files).